Portfolio/index.html

```diff
@@ -1,209 +1,236 @@
+            <div class="image-container">
+                <img src="/svelte.icon.png" alt="Sample Image" width="30px" height="30px">
+                <h2>Svelte</h2>
+            </div>
+            <div class="image-container">
+                <img src="/firebase.icon.png" alt="Sample Image" width="20px" height="20px">
+                <h2>Firebase</h2>
+            </div>
+            <div class="image-container">
+                <img src="/react.icon.png" alt="Sample Image" width="20px" height="20px">
+                <h2>React.js</h2>
+                <h2>React Native</h2>
+            </div>
 
-        <h4> <img src="/firebase.icon.png" alt="Css Image" width="30px" height="30px">
-            <div class="skilltext">Firebase</div>
-        </h4>
-        <h4> <img src="/react.icon.png" alt="Css Image" width="30px" height="30px">
-            <div class="skilltext">Reasctjs</div>
-            <div class="skilltext">React native</div>
-        </h4>
-    </section>
-    <section class="projects" id="projects">
-        <h1>Projects</h1>
 
-        <div class="project-text">
 
+        </div>
+    </section>
 
+    <section>
+        <div class="title">
+            <h1>Projects</h1>
+        </div>
+    </section>
+    <section class="projects" id="projects">
 
+        <div class="project1">
             <img src="/screen1.png" alt="css img" width="400px" height="400px">
-            <h1><b>Taskbot, Real Time Task Manager</b></h1>
-            <p>React.js, Firebase</p>
-            <p><b>Developed a dynamic task management application using React.js and Firebase.</b></p>
-            <p><b>Users can create, update, and delete tasks in time across devices. real-time across devices.</b></p>
-            <p><b>Integrated Firebase Authentication for secure user access.</b></p>
-            <div class="explore-btn">
-                <button>Explore</button>
-            </div>
-            <div class="project-text">
-
-
-
-                <img src="/screen2.png" alt="css img" width="400px" height="400px">
+            <div class="project-text1">
                 <h1><b>Taskbot, Real Time Task Manager</b></h1>
-                <p>React.js, typescript, Firebase</p>
-                <p><b>Developed a responsive e-commerce Website using React.js and Typescript.</b></p>
-                <p><b>Integrated Firebase for user authentication and firestore forproduct data storage.</b></p>
-                <p><b>Implemented secure payment processing with stripe API. </b></p>
-
+                <p>React.js, Firebase</p>
+                <p><b>Developed a dynamic task management application using React.js and Firebase.</b></p>
+                <p><b>Users can create, update, and delete tasks in time across devices. real-time across devices.</b>
+                </p>
+                <p><b>Integrated Firebase Authentication for secure user access.</b></p>
                 <div class="explore-btn">
                     <button>Explore</button>

```

Comment:
The 'filter' property for the image in the .image-container img class might need more adjustment for better visual clarity.